src/EditorConfigCompletionProvider.ts

```diff
@@ -4,14 +4,14 @@ import { TextDocument, Range, Position } from 'vscode'
 class EditorConfigCompletionProvider implements CompletionItemProvider {
 
 	private readonly properties: Property[] = [
-		new Property("root", ["true", "false"]),
-		new Property("charset", ["utf-8", "utf-8-bom", "utf-16be", "utf-16le", "latin1"]),
-		new Property("end_of_line", ["lf", "cr", "crlf"]),
-		new Property("indent_style", ["tab", "space"]),
-		new Property("indent_size", ["1", "2", "3", "4", "5", "6", "7", "8"]),
-		new Property("insert_final_newline", ["true", "false"]),
-		new Property("tab_width", ["1", "2", "3", "4", "5", "6", "7", "8"]),
-		new Property("trim_trailing_whitespace", ["true", "false"])
+		new Property("root", ["true", "false"], "Special property that should be specified at the top of the file outside of any sections. Set to true to stop .editorconfig files search on current file."),
+		new Property("charset", ["utf-8", "utf-8-bom", "utf-16be", "utf-16le", "latin1"], "Set to latin1, utf-8, utf-8-bom, utf-16be or utf-16le to control the character set. Use of utf-8-bom is discouraged."),
```

Comment:
Fair enough. I'm not sure why it's there either, but it's there! So you made a good decision 😄